<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" tests="5" skipped="0" failures="0" errors="0" timestamp="2020-05-29T13:59:52" hostname="bamboo-18" time="0.058">
  <properties/>
  <testcase name="negativeTimeout" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.001"/>
  <testcase name="timeoutExceptionThrownIfNoUserException" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.052"/>
  <testcase name="nullNextStatement" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.0"/>
  <testcase name="userExceptionPropagates" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.001"/>
  <testcase name="noExceptionThrownIfNoUserExceptionAndTimeoutDoesNotOccur" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
